tutorials/real-time-analytics-tutorial.rst

```diff
@@ -0,0 +1,153 @@
+::
+    
+    SELECT create_distributed_table('github_users', 'user_id');                                       
+    SELECT create_distributed_table('github_events', 'user_id');                               
+                                                                                          
+Sharding all tables on the user identifier allows Citus to :ref:`colocate <colocation>` these tables together,
+and allows for efficient joins and distributed roll-ups. You can learn more about the benefits of this approach `here <https://www.citusdata.com/blog/2016/11/29/event-aggregation-at-scale-with-postgresql/>`_.
+                                                                                          
+Then, you can go ahead and load the data we downloaded into the tables using the standard PostgreSQL :code:`\COPY` command.
+Please make sure that you specify the correct file path if you downloaded the file to a different location.
+
+::
+                                                                                          
+    \copy github_users from 'users.csv' with csv;                                                     
+    \copy github_events from 'events.csv' with csv;                                                     
+
+
+Running queries
+----------------
+
+Now that we have loaded data into the tables, let's go ahead and run some queries. First, let's check how many users we have in our distributed database.
+
+::
+                                                                                          
+    SELECT count(*) FROM github_users;
+    
+Now, let's analyze Github push events in our data. We will first find the number of distinct commits in push events and then compute the total number of commits by each minute.
+
+::
+                                                                                          
+    SELECT date_trunc('minute', created_at) AS minute,
+           sum((payload->>'distinct_size')::int) AS num_commits
+    FROM github_events
+    WHERE event_type = 'PushEvent'
+    GROUP BY minute
+    ORDER BY minute;                                                                                          
+
+We also have a users table. We can also easily join the users with events, and find users who created the most repositories. 
+
+::
+                                                                                          
+    SELECT login, count(*)
+    FROM github_events ge
+    JOIN github_users gu
+    ON ge.user_id = gu.user_id
+    WHERE event_type = 'CreateEvent' AND payload @> '{"ref_type": "repository"}'
+    GROUP BY login
+    ORDER BY count(*) DESC LIMIT 10;                                                                                          
+
+Citus also supports standard `INSERT`, `UPDATE`, and `DELETE` commands. For example, you can update a user's display login by running the following command:
```

Comment:
(Minor)

Citus also supports standard `INSERT`, `UPDATE`, and `DELETE` commands for ingesting and modifying your data.